package/yast2-registration.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Jan 24 15:29:29 UTC 2022 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Fix detection of base products coming from SCC (bsc#1194989).
```

Comment:
I think there is somewhere documentation for communication with scc. So we probably need to update it and ask scc if it was intentional